### DISCUSSION TITLE
Exporting results and issue with namespaces in file names

### DISCUSSION THREAD

**Discussion by gcelano** [korpling/ANNIS#880] [General]
When I export results in the CSV format, I get a large number of columns which I think are related to the way ANNIS interprets the namespaces in the file names [annis-export17078913905966896679.csv](https://github.com/user-attachments/files/17982116/annis-export17078913905966896679.csv). How can I avoid this issue? Should the name of each PAULA XML file start with the same namespace per kind of annotation across the entire corpus? Does this apply also to anno files (for example, `word_date.anno.xml`) and to the files containing the base text and the tokanization of it (for example, `text.ggm0001.ggm001.1st1K-grc1.xml` and `token.ggm0001.ggm001.1st1K-grc1.xml`)?

I

**Comment by thomaskrause** [comment]
The CSV exporter is quite limited in its configurability, e.g. it will always show the internal node ID. It will treat annotations with the same name but different namespaces always as different columns, and there is no configuration yet to change it. You can either unify the namespace or, if the users have enough local space and processing power, hint to the https://github.com/matthias-stemmler/annimate tool which provides much better export functionality for ANNIS.
